tests/schemas/root.json

```diff
@@ -0,0 +1,11 @@
+{
+    "type": "object",
+    "properties": {
+        "readme": {"type": "string"},
+        "features": {"type": "object"},
+        "unprotectedTemporary": {
+            "type": "array",
+            "items": {"type": "object"}
+        }
```

Comment:
what about "version"? BTW can we make validation warn us about unexpected fields?